webpages/settings/index.js

```diff
@@ -313,6 +313,11 @@ chrome.storage.sync.get(["globalTheme"], function ({ globalTheme = false }) {
           downloadBlob("scratch-addons-settings.json", blob);
         });
       },
+      copySettings() {
+        serializeSettings().then((serialized) => {
+          navigator.clipboard.writeText(serialized);
```

Comment:
I thought clipboard cannot be used on Promise callback?

Reply:
I don't know - I copied that clipboard code from somewhere. I mean, it works... Honestly, I don't even know that much JS.

Reply:
Well, the same code is used in forum-copy-code.

Reply:
There's no promise callback on forum-copy-code